### PR TITLE
make user component of card & user avatar of initials

### PR DESCRIPTION
**Pull request by Sooonne**
https://github.com/cybergizer-hq/retrospective/pull/new/user_without_avatar_has_initials_avatar

### CODE REVIEW

**Comment by Sooonne** [comment]
https://github.com/cybergizer-hq/retrospective/issues/89
https://github.com/cybergizer-hq/retrospective/issues/92
0. very small improvement in backend to get user first name & last name
1 make component card-user 
2 add functionality that if you dont have avatar, you will have circle with your initials
3  card-user component with name & surname now is on the top of the card (as was talked)
4. "drop by" (issue 92) is also here, coz it's just one dropped line and very small changes for separate task

File: app/javascript/components/ActionItem/index.js [hunk 1]
```diff
@@ -28,13 +29,24 @@ class ActionItem extends React.Component {
...
     const footerNotEmpty =
       movable || transitionable || timesMoved !== 0 || assignee !== null;
 
     return (
       <div className={`box ${this.pickColor()}_bg`}>
+        {assignee && (
+          <CardUser
+            avatar={avatar}
+            nickname={assignee}
+            fistsName={lastName}
```
Comment (IvanShamatov) [inline app/javascript/components/ActionItem/index.js:45]: `fistsName={lastName}` -- fiRst ну и да, логику тоже можно поправить

File: app/javascript/components/CardColumn/Card/card-user/card-user.jsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import {getUserInitials} from '../../../../utils/helpers';
+
+const CardUser = ({avatar, fisrtName, lastName, nickname}) => {
```
Comment (IvanShamatov) [inline app/javascript/components/CardColumn/Card/card-user/card-user.jsx:4]: какая-то шляпа с fiRSt

File: app/javascript/components/CardColumn/Card/index.js [hunk 3]
```diff
@@ -18,11 +21,16 @@ const Card = ({
 }) => {
   return (
     <div className="box">
+      <CardUser
+        avatar={avatar}
+        name={nickname}
+        fisrtName={firstName}
```
Comment (IvanShamatov) [inline app/javascript/components/CardColumn/Card/index.js:27]: fisrtName и тут fiRST нужно поправить

**Review by IvanShamatov — CHANGES_REQUESTED** [review]

**Comment by Sooonne** [comment]
Снимок экрана 2020-12-01 в 14 43 34
Снимок экрана 2020-12-01 в 14 41 03


Now is just one letter in initials because existing users don't have last name on server

**Review by IvanShamatov — APPROVED** [review]

**Pull request merged**